iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverterTest.java

```diff
@@ -0,0 +1,168 @@
+    public void buildObject_hbaseResult_NULL() throws Exception {
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			defaultApprover, relationColumnFamilyBytes);
+        converter.buildObject(null, oaf);
+    }
+    
+    @Test
+    public void buildObject_disapproved_candidates() throws Exception {
+        //given
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			new ResultApprover() {
+					@Override
+					public boolean approveBeforeBuilding(Oaf oaf) {
+						return false;
+					}
+				}, relationColumnFamilyBytes);
+    	
+    	String projectId = "someProjectId";
+    	String organizationId = "someOrgId";
+    	
+    	NavigableMap<byte[],byte[]> navigableMap = new TreeMap<byte[],byte[]>(
+    			UnsignedBytes.lexicographicalComparator());
+    	navigableMap.put(organizationId.getBytes(), createOafObject(projectId, organizationId).toByteArray());
+    	when(result.getFamilyMap(relationColumnFamilyBytes)).thenReturn(navigableMap);
+
+        // execute 
+        Collection<ProjectToOrganization> projOrgs = converter.buildObject(result, oaf);
+        
+        // assert
+        assertNotNull(projOrgs);
+        assertTrue(projOrgs.isEmpty());
+    }
+    
+    @Test
+    public void buildObject_no_candidates() throws Exception {
+        //given
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			defaultApprover, relationColumnFamilyBytes);
+    	
+    	when(result.getFamilyMap(relationColumnFamilyBytes)).thenReturn(new TreeMap<byte[],byte[]>(
+    			UnsignedBytes.lexicographicalComparator()));
+
+        // execute 
+        Collection<ProjectToOrganization> projOrgs = converter.buildObject(result, oaf);
+        
+        // assert
+        assertNotNull(projOrgs);
+        assertTrue(projOrgs.isEmpty());
+    }
+    
+    @Test
+    public void buildObject() throws Exception {
+        //given
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			defaultApprover, relationColumnFamilyBytes);
+    	
+    	String projectId = "someProjectId";
+    	String organization1Id = "someOrg1Id";
+    	String organization2Id = "someOrg2Id";
+    	
+    	NavigableMap<byte[],byte[]> navigableMap = new TreeMap<byte[],byte[]>(
+    			UnsignedBytes.lexicographicalComparator());
+    	navigableMap.put(organization1Id.getBytes(), createOafObject(projectId, organization1Id).toByteArray());
+    	navigableMap.put(organization2Id.getBytes(), createOafObject(projectId, organization2Id).toByteArray());
+    	when(result.getFamilyMap(relationColumnFamilyBytes)).thenReturn(navigableMap);
+
+        // execute 
+        Collection<ProjectToOrganization> projOrgs = converter.buildObject(result, oaf);
+        
+        // assert
+        assertNotNull(projOrgs);
+        assertEquals(2, projOrgs.size());
+        
+        Iterator<ProjectToOrganization> projOrgIt = projOrgs.iterator();
```

Comment:
If you returned list you could get elements by index and wouldn't have to use iterator.